jaba/src/pages/CreateUserPage/CreateUser.module.css

```diff
@@ -0,0 +1,250 @@
+body {
```

Comment:
Update the styling a bit to match the edits I made to PR 6 before I merged it in: 
https://github.com/Hack4Impact-UMD/jefferson-area-board-for-aging/pull/6/commits/f341aace57fa95bb84d0301bd62fb1dd6860c8f9

(Reorder CSS classes in file so closely related CSS classes are closer together in the file, remove duplicate and dead CSS, update CSS to be a bit more responsive)

Reply:
What do you mean by "update CSS to be a bit more responsive"?

Reply:
The updates do look better though!